packages/steps/src/utils/expodash/duplicates.ts

```diff
@@ -0,0 +1,19 @@
+import { uniq } from './uniq.js';
+
+export function duplicates<T = any>(items: T[]): T[] {
+  const uniqueItems = uniq(items);
+  if (uniqueItems.length === items.length) {
+    return [];
+  }
+
```

Comment:
```suggestion
```

An early exit would normally have performance befits, but most likely calling uniq is doing a similar amount of work as the rest of this function. Other than performance there is no benefit in keeping early exit code, it only complicates code by adding additional cases that regular implementation already handles correctly.